### PR TITLE
1) Errors revisited

### PR DESCRIPTION
**Pull request by jendakol**

### CODE REVIEW

File: internal/utils/http/http.go [hunk 1]
```diff
@@ -160,7 +151,20 @@ func SendEmptyResponse(w http.ResponseWriter, r *http.Request) {
...
+	logger := logging.FromContext(r.Context())
+
+	var mr, ok = err.(errors.ErouskaError)
+
+	if ok {
+		sendErrorResponse(w, r, mr.Code(), mr.Error())
+	} else {
+		logger.Warnf("Cannot handle request due to unknown error: %+v", err.Error())
+		sendErrorResponse(w, r, rpccode.Code_INTERNAL, err.Error())
+	}
```
Comment (marekaf) [inline internal/utils/http/http.go]: can this be if { .. return} without `else` pls? :D

**Comment by marekaf** [comment]
lgtm

File: internal/utils/http/http.go (hunk 1 above)
Comment (jendakol) [reply internal/utils/http/http.go]: As you wish :man_shrugging:

**Comment by marekaf** [comment]
so it this going to return status and message as requested in https://github.com/covid19cz/erouska-backend/issues/40#issuecomment-680760921 ?

**Comment by jendakol** [comment]
No, this is an independent thing. But yes, it could be done both at once, if you insist.

**Comment by marekaf** [comment]
> No, this is an independent thing. But yes, it could be done both at once, if you insist.

nope, I don't insist, just wanted to know:)

**Review by marekaf — APPROVED** [review]

**Pull request merged**